sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertSuite.scala

```diff
@@ -728,4 +732,254 @@ class InsertSuite extends QueryTest with TestHiveSingleton with BeforeAndAfter
       assert(e.contains("mismatched input 'ROW'"))
     }
   }
+
+  private def getConvertMetastoreConfName(format: String): String = format match {
+    case "parquet" => "spark.sql.hive.convertMetastoreParquet"
+    case "orc" => "spark.sql.hive.convertMetastoreOrc"
+  }
+
+  private def getSparkCompressionConfName(format: String): String = format match {
+    case "parquet" => "spark.sql.parquet.compression.codec"
+    case "orc" => "spark.sql.orc.compression.codec"
```

Comment:
Here, too.
- `SQLConf.PARQUET_COMPRESSION.key` instead of "spark.sql.parquet.compression.codec"
- `SQLConf.ORC_COMPRESSION.key` insead of "spark.sql.orc.compression.codec"